<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBLP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image:url(:/DS_2020_GUI/image/astronomy-atmosphere-earth-exploration.jpg)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>600</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url();</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="tet_inputParameter">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>100</y>
       <width>350</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbn_Info">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>175</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 20pt &quot;Consolas&quot;;</string>
     </property>
     <property name="text">
      <string>标题查询</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbn_Author">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>175</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 20pt &quot;Consolas&quot;;</string>
     </property>
     <property name="text">
      <string>作者查询</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbn_Title">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>175</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 20pt &quot;Consolas&quot;;</string>
     </property>
     <property name="text">
      <string>模糊查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>181</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/DS_2020_GUI/image/search.png);
color:rgb(255, 255, 255);
font: 75 28pt &quot;Consolas&quot;;</string>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>10</y>
       <width>350</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 28pt &quot;Consolas&quot;;</string>
     </property>
     <property name="text">
      <string>DBLP 数据查询</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_F5">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>-10</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/DS_2020_GUI/image/planet.png);
color:rgb(255, 255, 255);
font: 75 12pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>F5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_F2_TopAuthor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>125</width>
      <height>125</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/DS_2020_GUI/image/planet.png);
color:rgb(255, 255, 255);
font: 75 12pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>作者(Top100)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_F3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/DS_2020_GUI/image/planet.png);
color:rgb(255, 255, 255);
font: 75 12pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>历年热词</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_help">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>460</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/DS_2020_GUI/image/feather-pen.png);
color:rgb(255, 255, 255);
font: 75 12pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>help</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../DS_2020_GUI.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_btn_loadfile_clicked()</slot>
 </slots>
</ui>
